pages/osx/codesign.md

```diff
@@ -0,0 +1,11 @@
+# codesign
+
+> Sign code for the Mac certificate store.
+
+- Link a certificate to your application:
+
+`codesign -s "My Company Name" /path/to/App.app`
+
+- Verify a certificate to an application:
```

Comment:
`to an application` feels awkward. Do you mean `with an application`? Or as it that the application is being verified against the specified certificate?

Reply:
With this, the user sets an application to be 'theirs' by pairing it with the certificate.

Reply:
Well, in that case that is what the wording should be. "Verify a certificate" does not signify in any way that it will pair the certificate with the application.

Reply:
Sorry, I was referring to the other command. For the "Verify a certificate"... it verifies that an application's certificate is valid.